esp/src/eclwatch/ActivityWidget.js

```diff
@@ -187,6 +192,8 @@ define([
                 return;
 
             var context = this;
+            this.autoRefreshButton = registry.byId(this.id + "AutoRefresh");
+            this.activity.disableMonitor(!this.autoRefreshButton.get("checked"));
```

Comment:
On initialization we need to disableMonitoring which ActivityWidget.js does not do which is the point of this.activity.disableMonitor(!this.autoRefreshButton.get("checked")); We can also say it by: this.activity.disableMonitor(true); if it makes more sense.